app/services/oneoffs/flipflop_users_cleanup.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module Oneoffs
+  class FlipflopUsersCleanup
+    def call
+      result = []
+
+      flip_flop_users.each do |from_user|
+        if from_user.archived?
```

Comment:
So on re-reading this it looks like we take the `user_id` in the query, which is the `to_user` of the `ParticipantIdChange` - so `from_user` is actually the `to_user` here (but having it as `from_user` maybe makes sense as its the user we're potentially flipping away from?). 

If I've got that right, then surely we would want to continue and transfer away from them if they are archived? It's doesn't really matter either way because we're going to fix all those users in #4530 anyway, but I don't think the logic makes sense here?

Reply:
From the flip flop pov, its actually both from and to users. I guess `flip_flop_users` returns all affected users.

At this point, we're looping them and treating each one as the "from user" to determine if there is a "to user" (if primary, that means we shouldn't be transferring etc).

Reply:
I see that makes sense 👍